rialto_airflow/dags/harvest.py

```diff
@@ -164,33 +164,34 @@ def link_funders(snapshot, openalex_fill_in, dimensions_fill_in):
     def publish_openaccess(snapshot):
         openaccess.write_publications(snapshot)
         openaccess.write_contributions(snapshot)
+        openaccess.write_contributions_by_school(snapshot)
+        openaccess.write_contributions_by_department(snapshot)
 
     @task()
     def upload_publish_files(snapshot):
-        open_access_publications_filename = "publications.csv"
-        open_access_contributions_filename = "contributions.csv"
-        open_access_publications_file_id = google.get_file_id(
-            google.open_access_dashboard_folder_id(), open_access_publications_filename
-        )
-        open_access_contributions_file_id = google.get_file_id(
-            google.open_access_dashboard_folder_id(), open_access_contributions_filename
-        )
-
-        logging.info(
-            f"Uploading {snapshot.path / open_access_publications_filename} to file id {open_access_publications_file_id}"
-        )
-        google.replace_file_in_google_drive(
-            str(snapshot.path / open_access_publications_filename),
-            open_access_publications_file_id,
-        )
-
-        logging.info(
-            f"Uploading {snapshot.path / open_access_contributions_filename} to file id {open_access_contributions_file_id}"
-        )
-        google.replace_file_in_google_drive(
-            str(snapshot.path / open_access_contributions_filename),
-            open_access_contributions_file_id,
-        )
+        csv_files = [
+            "publications.csv",
+            "contributions.csv",
+            "contributions-by-school.csv",
+            "contributions-by-department.csv",
+        ]
+
+        google_folder_id = google.open_access_dashboard_folder_id()
+
+        for csv_file in csv_files:
+            file_path = snapshot.path / csv_file
+            google_file_id = google.get_file_id(google_folder_id, csv_file)
+
```

Comment:
Since the new files weren't on the server I needed to add this conditional to upload them when not already present in the drive.

Reply:
good idea, we could possibly refactor the google code to make a "add_or_replace" type method since this could be useful in other spots too

Reply:
Just did it here: https://github.com/sul-dlss/rialto-airflow/pull/367

We could do a follow on PR for this one if we merge this before the other